packages/@uppy/transloadit/src/index.js

```diff
@@ -14,9 +14,9 @@ function defaultGetAssemblyOptions (file, options) {
   }
 }
 
-const UPPY_SERVER = 'https://api2.transloadit.com/uppy-server'
-// Regex used to check if an uppy-server address is run by Transloadit.
-const TL_UPPY_SERVER = /https?:\/\/api2(?:-\w+)?\.transloadit\.com\/uppy-server/
+const COMPANION = 'https://api2.transloadit.com/companion'
+// Regex used to check if an companion address is run by Transloadit.
+const TL_COMPANION = /https?:\/\/api2(?:-\w+)?\.transloadit\.com\/companion/
```

Comment:
Would be nice to throw an error if anyone uses the old URL maybe. breakage because of this change would be very hard to debug for users I think

Reply:
Throw in the client (Transloadit plugin) if an 'uppy-server' URL is used rather than a 'companion' URL.

The breakage here is subtle: if users do `{ serverUrl: 'http://api2.transloadit.com/uppy-server' }`, they would get a sticky server before this patch, but may end up sending different requests to different servers after this patch.

For the time being we could also change the regex to allow `(companion|uppy-server)` and keep exporting `Transloadit.UPPY_SERVER` while removing it from the documentation.